--- func-0.18/setup.py~	2008-03-07 21:38:26.000000000 +0200
+++ func-0.18/setup.py	2008-05-23 22:13:37.603137505 +0300
@@ -16,7 +16,7 @@
         manpath    = "share/man/man1/"
         etcpath    = "/etc/%s" % NAME
         etcmodpath = "/etc/%s/modules" % NAME
-        initpath   = "/etc/init.d/"
+        initpath   = "/etc/rc.d/init.d/"
         logpath    = "/var/log/%s/" % NAME
 	varpath    = "/var/lib/%s/" % NAME
         rotpath    = "/etc/logrotate.d"
